src/tests/python/CMakeLists.txt

```diff
@@ -42,8 +42,8 @@ foreach(test ${tests})
       ENVIRONMENT "PYTHONHOME=${Python3_RUNTIME_LIBRARY_DIRS}")
   endif()
   # No need to set PYTHONPATH since we are not importing the package in any test here
-  #set_property(TEST ${test} APPEND PROPERTY
-  #  ENVIRONMENT "PYTHONPATH=${dlite_PYTHONPATH_NATIVE}")
+  set_property(TEST ${test} APPEND PROPERTY
```

Comment:
@jesper-friis Should this be commented out again?

Reply:
yes, I think so